src/kOS/Module/kOSProcessor.cs

```diff
@@ -82,7 +95,7 @@ public void Activate()
             OpenWindow();
         }
 
-        [KSPField(isPersistant = true, guiName = "Required Power", guiActive = true)]
+        [KSPField(isPersistant = true, guiName = "kOS Required Power", guiActive = true, guiActiveEditor = true)]
```

Comment:
Actually, I wrote a "Moving Average" class for use with the steering manager anyways, we could certainly use that class to keep track of the historical values.  In fact, that's what I do with it, to de-bounce the "observed" torque and the "update time".  I'll update to use a moving average.

I didn't notice your comment about title before committing "Max" but I like "Peak" better.  I'll probably plan on changing that.

Regarding your last point, while going through this, I decided I'd submit an issue for a "debug info" type class which could keep track of "instructions so far", "total instructions so far", variable count, trigger count, and any other information that we might be interested in knowing about operation.